hutch_python/tests/test_questionnaire.py

```diff
@@ -17,6 +17,7 @@ def clear_happi_cache():
 def test_qs_load():
     logger.debug('test_qs_load')
     hutch_python.qs_load.QSBackend = QSBackend
+    clear_happi_cache()
```

Comment:
pre-empting the question: clearing the cache here removed a red herring of a bug, helping my debugging process

Reply:
Interesting. A bug in the `happi_cache`?

Reply:
the happi_cache behaves as intended, the problem is that we're doing happi loads multiple times in one session, so if something is loaded incorrectly in one test but not in another, it may show up as an error in both of them- or the reverse, a bug might get masked!

Reply:
Make sense. There is a way to force the method to create a new device and overwrite the cache, but there is not a global option to do so. Wonder if there should be 🤔 ?